README.md

```diff
@@ -20,14 +20,34 @@ npm install react-transition-group --save
 yarn add react-transition-group
 ```
 
+## CDN / External
+
+To exclude the react-transition-group package from your primary build and use an external CDN when bundling, link
+to the following CDN:
+
+[https://unpkg.com/react-transition-group/dist/react-transition-group.min.js](https://unpkg.com/react-transition-group/dist/react-transition-group.min.js)
+
+In order to exclude react-transition-group using **Webpack**, include the following configuration in the `externals`
+webpack configuration section: 
+
+```
+externals: {
+    'react-transition-group': {
+        amd: 'react-transition-group',
+        commonjs2: 'react-transition-group',
+        commonjs: 'react-transition-group',
+        root: 'ReactTransitionGroup',
+    }
+```
+
 ## High-level API: CSSTransitionGroup
 
 `CSSTransitionGroup` is a high-level API based on [`TransitionGroup`](#low-level-api-transitiongroup) and is an easy way to perform CSS transitions and animations when a React component enters or leaves the DOM. It's inspired by the excellent [ng-animate](http://www.nganimate.org/) library.
 
 **Importing**
 
 ```javascript
-import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup' // ES6
+import { CSSTransitionGroup } from 'react-transition-group' // ES6
```

Comment:
Gotcha. Dropped the webpack build specific option above and added minor clarity. If you want to clarify the import or even better document both (Cherry picked for direct bundled or ES6 style import for those using /dist builds), fine with me.